src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java

```diff
@@ -10,8 +10,8 @@
 import net.minecraft.item.ItemGroup;
 
 /**
- * @author TehcJS
- * @author mounderfod
+ * Module for Taters! :>
```

Comment:
I don't think we need Javadoc for modules. I mainly made the Javadoc for things that seem more like "API"s than provide the real functionality. I. e. If we ever make the module system a separate library, that's what the Javadoc is for.